src/docs/deployment/fastlane-cd.md

```diff
@@ -28,10 +28,10 @@ delivery from a local machine.
     run `fastlane init`.
 1. Edit the `Appfile`s to ensure they have adequate metadata for your app.
     * ![Android](/images/fastlane-cd/android.png) Check that `package_name` in
-    `[project]/android/Appfile` matches your package in pubspec.yaml.
+    `[project]/android/Appfile` matches your package name in AndroidManifest.xml.
```

Comment:
instead of `package name in AndroidManifest.xml` it should be `applicationId in android/app/build.gradle` as i explained in the [issue comment](https://github.com/flutter/website/issues/2458#issuecomment-468940685), as they may be different